src/render/PointArray.cpp

```diff
@@ -154,8 +154,8 @@ static OctreeNode* makeTree(int depth, size_t* inds,
         return node;
     }
     // Partition points into the 8 child nodes
-    size_t* childRanges[9];
-    multi_partition(beginPtr, endPtr, OctreeChildIdx(P, center), childRanges+1, 8);
+    std::vector<size_t *> childRanges(9, NULL);
```

Comment:
Should be safe to assume std::array is available.  Personally I'm perfectly happy with a good old C array, provided you don't need the type preserved as it's passed around.